src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    private Movie movie;
+    @OneToOne
```

Comment:
it will be more correct to use ManyToOne because session could be only in one CinemaHall but CinemaHall could have a lot of different sessions in different time and day